_backend/users/models.py

```diff
@@ -47,7 +56,7 @@ class new_user_model(AbstractBaseUser, PermissionsMixin):
     objects = custom_account_manager_model()
 
     USERNAME_FIELD = 'email'
-    REQUIRED_FIELDS = ['user_name']
+    REQUIRED_FIELDS = ['user_name', 'first_name']
 
     # def levelcalculate (self):
```

Comment:
same thing as previous comment, anything not fully implemented should be removed right?